N/search.d.ts

```diff
@@ -160,6 +160,8 @@ export interface Search {
     runPaged: SearchRunPagedFunction;
 }
 
+import {FieldValue} from 'Record';
```

Comment:
Could you put the `import` at the top of the file to be consistent with the other uses of this pattern?